src/ansys/aedt/core/hfss.py

```diff
@@ -7087,18 +7087,22 @@ def wave_port(
     @pyaedt_function_handler()
     def plane_wave(
         self,
+        assignment=None,
         vector_format="Spherical",
         origin=None,
         polarization=None,
         propagation_vector=None,
         wave_type="Propagating",
         wave_type_properties=None,
         name=None,
-    ):
+    ) -> "BoundaryObject":
```

Comment:
```suggestion
    ) -> BoundaryObject:
```
I think this can be added without `""` since the class is imported at the beginning of the file.